app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -116,6 +116,10 @@ body {
     @include delta;
     line-height: 0.55;
   }
+
+  a {
```

Comment:
That's still not a good explanation to have two places. Is it just random? Are there no rules?